src/main/java/kevesse_kokanyolo_kod/menus/ProtoMenu.java

```diff
@@ -14,11 +16,18 @@
  *  ... 
  * 
  */
-public class ProtoMenu implements Printer {
-    public void println(String msg) {
-
+public class ProtoMenu {
+    Printer printer;
+    public ProtoMenu() {
```

Comment:
nem állít be semmit, akkor minek? Miért hívod meg a supert(), ha az őse csak implicite az Object

Reply:
Korábbi örökléses módszerből maradt, elfelejtettem törölni, de teljes mértékben jogos és javítottam